src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -137,10 +137,10 @@ private static void EncodingRegisterProvider()
         internal static SecurityZone GetFileSecurityZone(string filePath)
         {
             Diagnostics.Assert(Path.IsPathRooted(filePath), "Caller makes sure the path is rooted.");
-            Diagnostics.Assert(Utils.NativeFileExists(filePath), "Caller makes sure the file exists.");
+            Diagnostics.Assert(Utils.FileExists(filePath), "Caller makes sure the file exists.");
             string sysRoot = System.Environment.GetEnvironmentVariable("SystemRoot");
             string urlmonPath = Path.Combine(sysRoot, @"System32\urlmon.dll");
-            if (Utils.NativeFileExists(urlmonPath))
+            if (Utils.FileExists(urlmonPath))
```

Comment:
At least for these two uses, they do not expect exception to be thrown. So change them to `File.Exists` should be fine.

Reply:
Fixed.

Reply:
Maybe revert these two changes too, since we agreed to use a separate PR for the migration changes.

Reply:
Reverted.